contribs/gmf/src/services/authenticationservice.js

```diff
@@ -0,0 +1,247 @@
+
+
+/**
+ * Load the authentication service, which sends an asynch request to
+ * determine whether the user is currently connected or not.
+ * @private
+ */
+gmf.Authentication.prototype.load_ = function() {
+  var isLoggedInUrl = goog.uri.utils.appendPath(
+      this.baseUrl_, this.isLoggedInRouteSuffix_);
+  this.$http_.get(isLoggedInUrl, {
+    cache: false
+  }).then(
+      goog.bind(this.handleLogin_, this));
+};
+
+
+/**
+ * @param {string} newPwd
+ * @param {string} confPwd
+ * @return {angular.$q.Promise} Promise.
+ * @export
+ */
+gmf.Authentication.prototype.changePassword = function(newPwd, confPwd) {
+
+  var url = goog.uri.utils.appendPath(
+      this.baseUrl_, this.changePasswordRouteSuffix_);
+
+  // FIXME - use post, get is currently used for dev purpose
+  /*
+  return this.$http_.post(url, $.param({
+    newPassword: newPwd,
+    password: confirmNewPassword
+  }), {
+    headers: {'Content-Type': 'application/x-www-form-urlencoded'}
+  });
+  */
+  return this.$http_.get(url);
+};
+
+
+/**
+ * @param {string} login
+ * @param {string} pwd
+ * @return {angular.$q.Promise} Promise.
+ * @export
+ */
+gmf.Authentication.prototype.login = function(login, pwd) {
+
+  var url = goog.uri.utils.appendPath(this.baseUrl_, this.loginRouteSuffix_);
+
+  // FIXME - use post, get is currently used for dev purpose
+  /*
+  return this.$http_.post(url, $.param({login: login, password: pwd}), {
+    headers: {'Content-Type': 'application/x-www-form-urlencoded'}
+  }).then(goog.bind(
+  */
+  return this.$http_.get(url).then(
+      goog.bind(this.handleLogin_, this));
+};
+
+
+/**
+ * @return {angular.$q.Promise} Promise.
+ * @export
+ */
+gmf.Authentication.prototype.logout = function() {
+  var url = goog.uri.utils.appendPath(this.baseUrl_, this.logoutRouteSuffix_);
+  return this.$http_.get(url).then(
+      goog.bind(this.resetUser_, this));
+};
+
+
+/**
+ * @param {string} login
+ * @return {angular.$q.Promise} Promise.
+ * @export
+ */
+gmf.Authentication.prototype.resetPassword = function(login) {
+
+  var url = goog.uri.utils.appendPath(
+      this.baseUrl_, this.resetPasswordRouteSuffix_);
+
+  // FIXME - use post, get is currently used for dev purpose
+  /*
+  return this.$http_.post(url, $.param({login: login}), {
+    headers: {'Content-Type': 'application/x-www-form-urlencoded'}
+  }).then(goog.bind(
+  */
+  return this.$http_.get(url).then(goog.bind(
+      /**
+       * @param {angular.$http.Response} resp Ajax response.
+       * @return {gmf.AuthenticationDefaultResponse}
+       */
+      function(resp) {
+        var respData = /** @type {gmf.AuthenticationDefaultResponse} */ (
+            resp.data);
+        return respData;
+      }, this));
+};
+
+
+/**
+ * @param {angular.$http.Response} resp Ajax response.
+ * @return  {angular.$http.Response}
+ * @private
+ */
+gmf.Authentication.prototype.handleLogin_ = function(resp) {
+  var respData = /** @type {gmf.AuthenticationLoginResponse} */ (resp.data);
```

Comment:
It seems you didn't do that.
You need to call `isLoggedInRouteSuffix_` after a login, to retrieve more information.
Do you confirm that it returns more info than the login service ?